py3-dbs3-client.spec

```diff
@@ -0,0 +1,41 @@
+### RPM cms py3-dbs3-client 3.16.0
+## INITENV +PATH PYTHONPATH %i/${PYTHON_LIB_SITE_PACKAGES}
+## INITENV +PATH PYTHONPATH %i/x${PYTHON_LIB_SITE_PACKAGES}
+## INITENV SET DBS3_CLIENT_ROOT %i/
+## INITENV SET DBS3_CLIENT_VERSION %v
+## INITENV ALIAS dbs python $DBS3_CLIENT_ROOT/bin/dbs.py
+
+%define webdoc_files %{installroot}/%{pkgrel}/doc/
+Source0: git://github.com/dmwm/DBS.git?obj=master/%{realversion}&export=DBS&output=/%{n}.tar.gz
+Requires: python3 py3-cjson py3-dbs3-pycurl-client
+BuildRequires: py2-sphinx
```

Comment:
I think we should rely on a python3 sphinx spec.

Reply:
fixed now